R/extra_load_data_nonCollapsed.R

```diff
@@ -0,0 +1,87 @@
+#PURPOSE:
```

Comment:
Would have been neater to not duplicate the loading code - however, this makes the analysis entrypoint simpler to manage (i.e. use load_data.R or another extra script)!